config/routes.rb

```diff
@@ -1,8 +1,9 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: "pages#home"
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get "about", to: "pages#about"
+  get "my_profile", to: "pages#my_profile"
+
+  resources :venues
```

Comment:
And yes, as you mentioned I think reviews would be nested

Reply:
on it